internal/pkg/config/defaults/lte50_limits.yml

```diff
@@ -1,5 +1,6 @@
-ram:
-  max: 1024
+num_agents:
+  max: 50
+recommended_min_ram: 0
```

Comment:
No ram for 50 agents seems to be a bit too little?

Reply:
this effectively says there is no minimum, which matches up with the original limit calculation based on resources available. we could have a minimum for 0-50 agents, but there's nothing in the specs about an absolute minimum from what i saw.